pkg/node/local_node_store.go

```diff
@@ -18,6 +18,30 @@ type LocalNode struct {
 	// OptOutNodeEncryption will make the local node opt-out of node-to-node
 	// encryption
 	OptOutNodeEncryption bool
+	// hostEndpointID is the endpoint ID allocated for the host
+	hostEndpointID uint64
```

Comment:
To me this feels slightly the wrong place to put these. Wouldn't the endpoint manager be the more sensible place for this sort of information? It already has the `GetIngressEndpoint` anyway.

The `LocalNodeStore` was not meant for this sort of thing.

Reply:
Luckily I noticed that I had added these IngressEndpoint ID setter/getter to the node package without actually needing them. I removed them from this PR and left the host endpoint stuff as it was (using a global variable).